### ISSUE TITLE
Popover dividers are announced as menu items

### ISSUE THREAD

**Issue by robinmetral** [sumup-oss/circuit-ui#1660]
## Describe the bug

When using VoiceOver+Chrome on macOS, a Popover divider is announced as a menu item like other actions

## Steps to reproduce

1. Go to [the Popover story](https://circuit.sumup.com/?path=/story/components-popover--base)
2. Toggle the Popover with VoiceOver on (reproduced on Chrome, haven't tested other browsers or screen readers)
3. Hear the screen reader announcing "1 of 4 menu items" when there are only 3
4. (👉 this is an assumption) the 4th element that VoiceOver thinks is a menu item is the divider

## Expected behavior

VoiceOver should announce only the number of menu items as there are _actions_, excluding any dividers.

## Specifications

* Version: latest
* Browser: Chrome
* Screen Reader: VoiceOver
* OS: macOS Monterey

## Additional context

Some thoughts:
- Potential cause: although only actions have `role="menuitem"`, the wrapper has `role="menu"` and VO might assume that all children are menu items
- Potential fixes: `aria-hidden` or `role="presentation"` on divider children
